src/main/io/gps.c

```diff
@@ -1232,6 +1248,17 @@ void gpsConfigureUblox(void)
                             ubloxSetMessageRate(CLASS_NAV, MSG_NAV_DOP, 1);
                         }
                         break;
+                    case UBLOX_SAT_INFO:
+                        // enable by default, turned off when armed and receiving data to reduce in-flight traffic
+
+                        if (isConfiguratorConnected()) {
```

Comment:
This delay needs to go where it originally was located.
It served two functions:
- first, to give a second for the newly acquired baud rate in the module to stabilise
- second, to give time for Configurator to open and be detected as connected, reliably.
If the MSP method is reliable and will work with older configurator versions, then only 1s is needed.